spec/controllers/scrapers_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe ScrapersController, type: :controller do
+
+	describe "GET #index" do
+		xit "lists all scrapers" do
+			scraper = Scraper.create
+      get :index
+      expect(assigns(:scrapers)).to eq([scrapers])
```

Comment:
I think you just need to say `eq([scraper])` (without the `s`) to fix this test.